index.js

```diff
@@ -0,0 +1,230 @@
+let itemId = 1;
+let hundredArticles = [];
+
+
+// adds a listener to the search submit button
+const formListener = () => {
+	const formElement = document.querySelector("form");
+	const formInput = document.querySelector("#form-search");
+	formElement.addEventListener('submit', event => {
+		event.preventDefault();
+		search = formInput.value;
+		page=1;
+		getArticles(1,excludedDomainsArray);
+	});
+}
+
+// api request to news api. Returns json and calls createArticles function
+const getArticles = (page=1, excludedDomainsStr='', filterDomains="") => {
+	console.log('Function : Fetching Articles (getArticles)');
+	var url = 'https://newsapi.org/v2/everything?' +
+		'q=' + search  +
+		'&page=' + page +
+		'&apiKey=280f7af9f5c448c4a3598861960c947a' + 
+		'&excludeDomains=' + excludedDomainsStr + '&domains=' + filterDomains + '&pageSize=100&sortBy=publishedAt&language=en';
+	var req = new Request(url);
+	fetch(req)
+		.then(function (response) {
+			return response.json();
+		})
+		.then(function (body) {
+			hundredArticles = body.articles;
+			createArticles(0);
+			createExcluded();
+			publisherCount(body.articles);
+			addPagination(body.totalResults);
+
+		})
+}
+
+
+// takes request body and turns it into html to be appended into '.articles'
+const createArticles = (startArticle) => {
+	console.log('Function : Creating Articles (createArticles)');
+	clearDiv('.articles');
+	// console.log(hundredArticles);
+
+	twentyArticles = hundredArticles.slice(startArticle,startArticle+20);
+	itemId = 0;
+	twentyArticles.forEach(article => {
+		itemId ++
```

Comment:
you can get the itemId using forEach index for example `twentyArticles.forEach((article, index)){`